### ISSUE TITLE
Implementation details and code release schedule

### ISSUE THREAD

**Issue by kth0522** [liangsusan-git/AV-NeRF#2]
First of all, I would like to thank you for sharing the useful dataset. 

I have a few questions that I hope you can help clarify:

1. Audio Clip Length for Training
I may have missed it, but I couldn't find specific details regarding the audio clip length used for data processing in the training phase. Could you provide some information on this?

2. NeRFStudio Configuration
In the paper, it's mentioned that V-NeRF utilizes the Nerfacto model. However, the models provided by NeRFStudio include variants like Nerfacto, Nerfacto-Big, and Nerfacto-Huge. Could you specify which Nerfacto model was employed in your project?

3. Code Release Schedule
 Lastly, I am eager to know the planned schedule for the release of the model's code. I believe having access to the code would naturally address the above questions.

Thank you.

**Comment by swapb94** [comment]
+1

**Comment by liangsusan-git** [comment]
Hi,

1. Audio clip length: I downsample the video to 1 fps and set the length of the audio clip to 1 second. I set the `n_fft` as 512, `win_length` as 512, and `hop_length` as 128 when converting waveform audio to spectrogram using STFT.

2. Nerfstudio: I use the Nerfacto model as the Visual NeRF. I keep the default configurations for training, except I set `max-num-iterations` to 100,000.

3. Code release: I agree with you. I plan to release the code before February 15th. I know this is not ideal, but unfortunately, I cannot set aside time to release my code now.
